### DISCUSSION TITLE
How to show hidden key frames

### DISCUSSION THREAD

**Discussion by Tarkusb** [keiwando/toonsquid#55] [General]
Hi there, loving Toonsquid but how on earth do you get hidden keyframes to show up again?
I’ve looked through the online manual, not sure if I’m missing something 🙂

**Comment by Tarkusb** [comment]
Ok so worked this out now, the problem was a grouped layer, had to click on the individual drawing layer to find the keyframes. All that was showing when clicking on the layer in the timeline was an opacity property!
So all sorted.

**Reply by keiwando** [reply]
Also, just for reference, if you ever manually hide all properties of a layer in the timeline using the little "X" buttons, you can tap the animation layer and then select `Show all properties` which will show all of the properties again.

**Reply by Tarkusb** [reply]
Brilliant, thanks for that!
